js/footer.js

```diff
@@ -0,0 +1,16 @@
+var fifooter = "<div id='footer'> \
```

Comment:
There still is a duplicate ID, they must be unique.

Why not just add the `a` content in the last cell of the `#footer` with a `<br/>`?